.github/workflows/release-binary.yml

```diff
@@ -0,0 +1,80 @@
+          go build -o cql-proxy
+          tar cvfz cql-proxy-linux-amd64-${{ github.ref_name }}.tgz cql-proxy LICENSE
+      - name: Build Windows/amd64 binary
+        run: |
+          apt update
+          apt -y install zip
+          export GO111MODULE=on
+          export CGO_ENABLED=0
+          export GOOS=windows
+          export GOARCH=amd64
+          go build -o cql-proxy.exe
+          zip -vr cql-proxy-windows-amd64-${{ github.ref_name }}.zip cql-proxy.exe LICENSE
+      - name: Build Darwin/amd64 binary
+        run: |
+          export GO111MODULE=on
+          export CGO_ENABLED=0
+          export GOOS=darwin
+          export GOARCH=amd64
+          go build -o cql-proxy
+          tar cvfz cql-proxy-darwin-amd64-${{ github.ref_name }}.tgz cql-proxy LICENSE
+      - name: Build Darwin/arm64 binary
+        run: |
+          export GO111MODULE=on
+          export CGO_ENABLED=0
+          export GOOS=darwin
+          export GOARCH=arm64
+          go build -o cql-proxy
+          tar cvfz cql-proxy-darwin-arm64-${{ github.ref_name }}.tgz cql-proxy LICENSE
+      - name: Build Linux/arm64 binary
+        run: |
+          export GO111MODULE=on
+          export CGO_ENABLED=0
+          export GOOS=linux
+          export GOARCH=arm64
+          go build -o cql-proxy
+          tar cvfz cql-proxy-linux-arm64-${{ github.ref_name }}.tgz cql-proxy LICENSE
+      - name: Generate Checksums
+        run: |
+          sha256sum cql-proxy-linux-amd64-${{ github.ref_name }}.tgz | cut -d ' ' -f 1 > cql-proxy-linux-amd64-${{ github.ref_name }}-sha256.txt
+          sha256sum cql-proxy-windows-amd64-${{ github.ref_name }}.zip | cut -d ' ' -f 1 > cql-proxy-windows-amd64-${{ github.ref_name }}-sha256.txt
+          sha256sum cql-proxy-darwin-amd64-${{ github.ref_name }}.tgz | cut -d ' ' -f 1 > cql-proxy-darwin-amd64-${{ github.ref_name }}-sha256.txt
+          sha256sum cql-proxy-darwin-arm64-${{ github.ref_name }}.tgz | cut -d ' ' -f 1 > cql-proxy-darwin-arm64-${{ github.ref_name }}-sha256.txt
+          sha256sum cql-proxy-linux-arm64-${{ github.ref_name }}.tgz | cut -d ' ' -f 1 > cql-proxy-linux-arm64-${{ github.ref_name }}-sha256.txt
+      - name: Create Release
+        uses: softprops/action-gh-release@v1
+        with:
+          name: ${{ github.ref_name }}
+          files: |
+            cql-proxy-linux-amd64-${{ github.ref_name }}.tgz
+            cql-proxy-linux-amd64-${{ github.ref_name }}-sha256.txt
+            cql-proxy-windows-amd64-${{ github.ref_name }}.zip
+            cql-proxy-windows-amd64-${{ github.ref_name }}-sha256.txt
+            cql-proxy-darwin-amd64-${{ github.ref_name }}.tgz
+            cql-proxy-darwin-amd64-${{ github.ref_name }}-sha256.txt
+            cql-proxy-darwin-arm64-${{ github.ref_name }}.tgz
+            cql-proxy-darwin-arm64-${{ github.ref_name }}-sha256.txt
+            cql-proxy-linux-arm64-${{ github.ref_name }}.tgz
+            cql-proxy-linux-arm64-${{ github.ref_name }}-sha256.txt
```

Comment:
Yuck... it's kinda unpleasant to have to list every option explicitly here.  If you create a subdirectory and then create all your tarballs and checksum files in that directory you should just be able to use a single glob here ("with.files: subdir/*" rather than having to list everything out like this.  The [docs](https://github.com/softprops/action-gh-release?tab=readme-ov-file#%EF%B8%8F-uploading-release-assets) for action-gh-release indicates that globs are supported but obviously you should test that out.

Reply:
I think I have fixed this part but it is hard to test locally. AFAIK the only way would be to merge the PR and create old tag (e.g. 0.0.1) to see if upload works. We can remove the test release afterwards.

Reply:
Fixed, tested on my fork and it works.